src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -151,10 +151,10 @@ public function __construct(QueryBuilder $query)
     /**
      * Create and return an un-saved model instance.
      *
-     * @param  array  $attributes
+     * @param  array|\Illuminate\Support\ValidatedInput  $attributes
      * @return \Illuminate\Database\Eloquent\Model|static
      */
-    public function make(array $attributes = [])
+    public function make($attributes = [])
```

Comment:
Union types are supported as of PHP 8.0. Why not supply union typing?

```suggestion
    public function make(array|ValidatedInput $attributes = [])
```

Reply:
Absolutely open to this. I wasn't sure how much of the framework is typed, especially with union types. So I didn't do this.

Also, it's worth noting that not all methods which access "attributes" were type hinted with `array`. In the end, felt it's best to let the core team decide as this is an easy change.